lib/widgets/navigation_bar/custom_tab_bar.dart

```diff
@@ -14,23 +14,70 @@ class CustomTabBar extends StatelessWidget {
         : screenWidth > 1100
             ? 0.4
             : 0.45;
-    return Padding(
-      padding: EdgeInsets.only(left: screenWidth * 0.05),
-      child: SizedBox(
-        width: screenWidth * tabBarScaling,
-        child: Theme(
-          data: ThemeData(
-            highlightColor: Colors.transparent,
-            splashColor: Colors.transparent,
-            hoverColor: Colors.transparent,
+    return Container(
```

Comment:
Small design thing but is it possible to make "Creator's Corner" the same font size as "My Workouts" and "Explore"?

Reply:
We added a ticket for this because it's a scaling issue that I haven't been able to fix.